pkg/extension/templates/applicationclass.yaml

```
@@ -0,0 +1,30 @@
+#applicationClassGroup: mysql
+#name: mysql-b
+#provisioner: kubesphere.io/helm-application
+#appVersion: 8.0.0
+#packageVersion: 1.0.2
+#description:
+#  zh: 示例ApplicationClass
+#  en: Sample ApplicationClass
+#maintainer:
+#  name: demo
+#  email: demo@demo.com
+#icon: ./favicon.svg
```

We now put the icon in the static directory

```suggestion
#icon: ./static/favicon.svg
```